<launch>
  <!-- Where to find intrinsics, extrinsics, parameters -->
  <arg name="dataset_name" value="RealSenseIR"/>
  <arg name="online" default="false" />  
  <arg name="use_lcd" default="true" />  
  <arg name="rosbag_path" value="/home/andrew/Documents/SPARK/RealSense/Church/PianoOrganSynchronized.bag"/>
  <arg name="verbosity"    default="0" />
  <arg name="viz_type"     default="0" />

  <arg name="log_output" default="true"/>
  <arg name="log_output_path"
       default="$(find kimera_vio_ros)/output_logs/PianoOrganExperimental"
       if="$(arg log_output)"/>
  <arg name="log_gt_data" default="false"/>

  <!-- Subscribed Topics -->
  <arg name="left_cam_topic"  value="/vio_camera/infra1/image_rect_raw"/>
  <arg name="right_cam_topic" value="/vio_camera/infra2/image_rect_raw"/>
  <arg name="imu_topic"       value="/vio_camera/imu"/>

  <!-- Frame IDs for Odometry -->
  <arg name="base_link_frame_id" value="kimera_base_link" />

  <!-- Launch actual pipeline -->
  <include file="$(find kimera_vio_ros)/launch/kimera_vio_ros.launch"
           pass_all_args="true"/>
</launch>
